src/main.ts

```diff
@@ -626,387 +315,98 @@ async function main() {
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-  gridRight().ontouchstart = async () => {
-    if (isMoving) return
-    styleMove('move')
-    isMoving = true
-    useTouch = true
-    let success = await quadrantMoveHander(0,1)
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-
-  gridFrontLeft().onmousedown = async () => {
-    if (isMoving) return
-    if (useTouch) return
-    styleMove('move')
-    isMoving = true
-    let success = await quadrantMoveHander(1,-1)
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-  gridFrontLeft().ontouchstart = async () => {
-    if (isMoving) return
-    styleMove('move')
-    isMoving = true
-    useTouch = true
-    let success = await quadrantMoveHander(1,-1)
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-
-  gridHome().onmousedown = async () => {
-    if (isMoving) return
-    if (useTouch) return
-    styleMove('move')
-    isMoving = true
-    let success = await homeHandler();
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-  gridHome().ontouchstart = async () => {
+  async function touchStart(func: () => Promise<boolean>) {
     if (isMoving) return
     styleMove('move')
     isMoving = true
     useTouch = true
-    let success = await homeHandler();
+    let success = await func()
     if (success) {
       styleMove('ready')
       isMoving = false
     }
   };
   
-  async function homeHandler() {
-    try {
-      await home(motionClient, armClient);
-    } catch (error) {
-      console.log(error);
-      styleMove('error')
-      setTimeout( () => { styleMove('ready'); isMoving = false; }, 3000 )
-      return false
-    }
-    return true
+  function setButtonBehavior(button: HTMLTableCellElement, func: () => Promise<boolean>) {
+    button.onmousedown = async () => {mouseDown(func)}; 
+    button.ontouchstart = async () => {touchStart(func)};
   }
 
-  gridFrontRight().onmousedown = async () => {
-    if (isMoving) return
-    if (useTouch) return
-    styleMove('move')
-    isMoving = true
-    let success = await quadrantMoveHander(1,1)
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-  gridFrontRight().ontouchstart = async () => {
-    if (isMoving) return
-    styleMove('move')
-    isMoving = true
-    useTouch = true
-    let success = await quadrantMoveHander(1,1)
-    if (success) {
-      styleMove('ready')
-      isMoving = false
-    }
-  };
-
-  async function quadrantMoveHander(x,y) {
+  // Define buttons for imcremental movement in plane
+  async function planarMoveHandler(button: HTMLTableCellElement, x:number, y: number) {
     try {
-      await moveToQuadrant(motionClient, armClient, x, y);
+      await inPlaneMove(motionClient, armClient, x, y);
+      if (button.classList.contains('custom-box-shadow-active')) {await planarMoveHandler(button, x, y)};
```

Comment:
I think this doesn't actually do anything in the new design, but we can remove later

Reply:
Yeah I was wondering about that.  If you want I can remove now